### DISCUSSION TITLE
Copilot stopped suggesting

### DISCUSSION THREAD

**Discussion by Wombosvideo** [community/community#11411] [Copilot Conversations] [answered]
Since a few updates Github Copilot no longer suggests anything in VSCode. The only output I get from copilot is
```[INFO] [solutions] [<thedate>] prompt: {"prefix":"// Language: javascript\n// Path: src/assets/js/test.js\nfunction calculateDaysBetweenDates(begin, end) {}","suffix":""}```
But it doesn't suggest anything. `CTRL + Enter` shows 0 solutions. Copilot is enabled for all file types afaik.

Some version numbers:
Ubuntu 21.10 (impish)
VSCode 1.64.2-1644445741
Copilot v1.7.5011 (nightly)

Tested reinstalling VSCode and Copilot and removing the `~/.vscode/extensions/github.copilot-*` directory without any luck.

**Comment by davecheney** [comment]
I’m sorry you’re having difficulties. I believe copilot is working, perhaps you could try some other examples, or try copilot on a codebase that you’re working on.

**Reply by Wombosvideo** [reply]
I tried several different projects and programming languages. Around the time I started writing LaTeX in VSCode one or two months back I first noticed the issue. Copilot was working perfectly fine before that so at first I just assumed that copilot lacks LaTeX support which apparently is not the case. Is it technically possible that there are conflicting extensions?

**Reply by davecheney** [reply]
> Is it technically possible that there are conflicting extensions?

Very. It is common for extensions to compete for common keys like tab for completion.

**Reply by Wombosvideo** [reply]
Okay yeah, I get that but it's not even _showing_ any suggestions and blindly pressing `TAB` doesn't do a thing either.

Is there any way to enable a special Debug mode that tells you what's actually going on behind the scenes? The Output windows barely shows anything which kind of makes it hard to find the cause of my problem.

Also, I disabled most of my extensions and it still won't show any "solutions".

**Reply by davecheney** [reply]
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by Wombosvideo** [reply]
You say "relevant parts" but there are none, so I'll just paste the whole log (replacing timestamps with `<thedate>`):
```
[INFO] [ghostText] [<thedate>] Cancelled after extractPrompt
[INFO] [solutions] [<thedate>] prompt: {"prefix":"// Language: javascript\n// Path: src/assets/js/test.js\n// countdown from 30s to 0\nlet timer = 30;\nlet interval = 1;","suffix":""}
```

From my point of view this absolutely doesn't help to reproduce the error nor find the cause. Also, my roommate can use Copilot without any problems, also using a Linux distro. So the problem should not be network-related.

**Reply by davecheney** [reply]
That’s quite odd, normally copilot is rather verbose, they’re should be multiple lines, basically every few ms as you are typing.

**Reply by Wombosvideo** [reply]
Yeah that's also what I remember from back in the days Copilot was working fine. The only thing I did since then which could have broken Copilot is... updating it. And updating VSCode.

Either way, explaining code with "Labs" works but suggesting code in the editor doesn't -- again, "Synthesizing 0/10 solutions". By the way, is it normal that `https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions` returns a 404 when opening it in the browser?

**Reply by davecheney** [reply]
Yes, that url only responds to post

**Comment by Wombosvideo** [comment, marked as answer]
Apparently Github Copilot conflicts with the [Vscodefy extension](https://marketplace.visualstudio.com/items?itemName=iagolaguna.vscodefy). This makes absolutely no sense to me but after disabling it and reloading VSCode Github Copilot once again started suggesting code. As soon as I enable it again it immediately stops suggesting code.

**Steps to fix Github Copilot:**
1. Disable the extension Vscodefy
2. Reload Visual Studio Code

An issue on Vscodefy has also been filed, see [here](https://github.com/iagolaguna/vscodefy/issues/28)

**Reply by dlbone** [reply]
@Wombosvideo You are my savior thank you for this!!!

**Reply by ShaharMS** [reply]
You just saved me a **BUNCH** of hours - thanks :D
